openshift_cli_installer/tests/all_rosa_versions.py

```diff
@@ -0,0 +1,319 @@
+BASE_AVAILABLE_VERSIONS_DICT = {
```

Comment:
is there a dynamic way to fetch all rosa versions?

Reply:
This is for the tests, why would I want this to be dynamic?

Reply:
so that we won't need to update new releases manually